Project/Source/Resources/ResourceClip.cpp

```diff
@@ -30,7 +30,7 @@
 #define JSON_TAG_LOOP "Loop"
 #define JSON_TAG_SPEED "Speed"
 #define JSON_TAG_ID "Id"
-#define JSON_TAG_DURATION_BY_KEYFRAMES_SIZE "DurationByKeyframesSize"
+#define JSON_TAG_FRAMERATE "DurationByKeyframesSize"
```

Comment:
The JSON string is not the same as the tag

Reply:
Done